SECURITY.md

```diff
@@ -0,0 +1,36 @@
+
+## Security Announcements
+
+Join the [kubestellar-security-announce](https://groups.google.com/u/1/g/kubestellar-security-announce) group for emails about security and major API announcements.
+
+## Report a Vulnerability
+
+We're extremely grateful for security researchers and users that report vulnerabilities to the KubeStellar Open Source Community. All reports are thoroughly investigated by a set of community volunteers.
+
+You can also email the private [kubestellar-security-announce@googlegroups.com](mailto:kubestellar-security-announce@googlegroups.com) list with the security details and the details expected for [all KubeStellar bug reports](https://github.com/kcp-dev/edge-mc/blob/main/.github/ISSUE_TEMPLATE/bug_report.yaml).
```

Comment:
"announce" sounds wrong for the ingress of problem reports, "announce" suggests this is for egress from the maintainers to the public.

Reply:
I used the same naming convention as Kubernetes

Reply:
I think not.  Compare https://kubernetes.io/docs/reference/issues-security/security/#security-announcements with https://kubernetes.io/docs/reference/issues-security/security/#report-a-vulnerability

Reply:
yes - so here is my thinking on that one - there was a whole bunch of structure around a security response committee (SRC) - which I was not ready to commit to at this time.  So I made it simple, for now.  Aspiration is to get there, when we get a bit more resourced.

Reply:
Sure.  I got the simplification.  My only point is that the term "announce" is a confusing choice for an ingress point.